docs/manuals/core/entities/scenario-creation.md

```diff
@@ -77,7 +77,7 @@ Three parameters can be given to the scenario creation method :
 !!! Example
 
     === "Scenario creation with parameters"
-        Using the [`my_config.py`](../my_config.py) module, here is an example of how to create a scenario.
+        Using the [`my_config.py`](https://github.com/Avaiga/taipy-doc/blob/develop/docs/manuals/core/my_config.py) module, here is an example of how to create a scenario.
```

Comment:
We cannot have an absolute link for that. Here you are pointing to the develop version of the file which might change over time and over the versions.